src/main/java/nextstep/courses/domain/ImageType.java

```diff
@@ -0,0 +1,24 @@
+package nextstep.courses.domain;
+
+import java.util.Arrays;
+
+public enum ImageType {
+    GIF("gif"),
+    PNG("png"),
+    JPEG("jpeg"),
+    JPG("jpg"),
+    SVG("svg");
+
+    private String type;
+
+    ImageType(String type) {
+        this.type = type;
+    }
+
+    public ImageType getType(String type) {
+        return Arrays.stream(values())
+                .filter(imageType -> imageType.type.equals(type))
```

Comment:
```suggestion
                .filter(it -> it.sameType(type))
```

객체에게 물어보기를 의식적으로 훈련해보아요~

Reply:
객체에게 물어보는 것이 계속 노력해도 잘 안되는 부분입니다. ㅠ 
의식적으로 계속 노력해보겠습니다.